src/main/java/subway/line/web/LineController.java

```diff
@@ -43,9 +47,22 @@ public ResponseEntity<LineResponse> showLine(@PathVariable Long lineId) {
     public void updateLine(@PathVariable Long lineId, @RequestBody LineRequest lineRequest) {
         lineService.updateLine(lineId, lineRequest);
     }
+
     @DeleteMapping("/{lineId}")
     public ResponseEntity<Void> deleteLine(@PathVariable Long lineId) {
         lineService.deleteLine(lineId);
         return ResponseEntity.noContent().build();
     }
+
+    @PostMapping("/{lineId}/sections")
+    public ResponseEntity<Void> createSection(@PathVariable Long lineId, @RequestBody SectionRequest sectionRequest) {
+        lineSectionService.saveSection(lineId, sectionRequest);
+        return ResponseEntity.created(URI.create("/lines/" + lineId)).build();
```

Comment:
```suggestion
        return ResponseEntity.created(URI.create("/lines/" + lineId + "/sections")).build();
```

이런형태가 더 어울릴것 같습니다 😄 

[참고](https://developer.mozilla.org/ko/docs/Web/HTTP/Headers/Location)